wdl-engine/src/stdlib/size.rs

```diff
@@ -30,61 +30,6 @@ use crate::stdlib::ensure_local_path;
 /// The name of the function defined in this file for use in diagnostics.
 const FUNCTION_NAME: &str = "size";
 
-/// Gets the size of a local file.
```

Comment:
These functions were moved below simply to have the `size` function (the one implementing the WDL stdlib `size` function) be the first in the file.